### PR TITLE
Setting dynamic trending section filtering posts with tag trending #54

### PR DESCRIPTION
**Pull request by mmavroudi**

### CODE REVIEW

File: projects/views.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def home(request):
     culture_posts = posts.filter(categories__slug='culture')
     business_posts = posts.filter(categories__slug='business')
     lifestyle_posts = posts.filter(categories__slug='lifestyle')
+    post_trending = posts.filter(tags__name__in=["trending"])
```
Comment (kkampardi) [inline projects/views.py]: We need to render the last 6 posts tagged as 'trending' not all of them

File: projects/templates/home.html [hunk 2]
```diff
@@ -114,41 +114,15 @@ <h2><a href="{% url 'single_post' slug=post.slug %}">{{ post.title }}</a></h2>
                 <div class="trending">
                   <h3>Trending</h3>
                   <ul class="trending-post">
+                    {% for post in post_trending %}
                     <li>
                       <a href="single-post.html">
-                        <span class="number">1</span>
-                        <h3>The Best Homemade Masks for Face (keep the Pimples Away)</h3>
+                        <span class="number">{{ forloop.counter }}</span>
```
Comment (kkampardi) [inline projects/templates/home.html:120]: Nice!

**Review by kkampardi — CHANGES_REQUESTED** [review]
Very good progress. Lets learn how to query the last n records in django 
See the comment below

**Comment by mmavroudi** [comment]
I need to take the changes in model Post from pull request #57 to continue editing here for latest trending 6 posts.
I need you to merge #57.

**Review by kkampardi — APPROVED** [review]
Looks good!

**Pull request merged**